### PR TITLE
Propose Goal Code: Human Readable Verifiable Identifier

### PR DESCRIPTION
**Pull request by TelegramSam**
Signed-off-by: Sam Curren <telegramsam@gmail.com>

### CODE REVIEW

File: concepts/0530-goal-human-readable-verified-identifer/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+A Verifiable identifier requires that some level of assurance is provided that control has been proven over the identifier. 
+
+Governance Frameworks that serve this goal code must provide details about which identifiers are acceptable, and how they are to be verified.  
+
+## Governance Frameworks 
+
+The following Governance Frameworks are related to this goal code.
+
+- Domain Control 
+- Email Ownership
```
Comment (llorllale) [inline concepts/0530-goal-human-readable-verified-identifer/README.md:41]: These two frameworks are not documented anywhere that I know of.

**Comment by TelegramSam** [comment]
Readable -> Friendly

**Review by kdenhartog — APPROVED** [review]
The first thing that came to mind for how to implement this is using some sort of identity-based cryptosystem where the identifier maps to the public key. I'm curious to see which direction it goes.

File: concepts/0530-goal-human-readable-verified-identifer/README.md (hunk 1 above)
Comment (TelegramSam) [reply concepts/0530-goal-human-readable-verified-identifer/README.md:41]: Haven't written those yet.

**Comment by TelegramSam** [comment]
Discussed 20241009. Good to keep as a documented concept though not implemented anywhere.

**Pull request merged**